app/code/Magento/Customer/Controller/Address/FormPost.php

```diff
@@ -129,17 +162,35 @@ protected function _extractAddress()
     /**
      * Retrieve existing address data
      *
+     * @param CustomerInterface|null $customer
      * @return array
      * @throws \Exception
      */
-    protected function getExistingAddressData()
+    protected function _getExistingAddressData(CustomerInterface $customer = null): array
```

Comment:
Unfortunately, we can't change public and protected methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
the same here, `getExistingAddressData` is original one `_getExistingAddressData` was added